### ISSUE TITLE
Tested and working with Ubuntu 16.04 (xenial)

### ISSUE THREAD

**Issue by kmpm** [ellysh/mxu11x0#3]
Hi,
Just wanted to inform that I could use this driver in Ubuntu Xenial.
Only tested with `screen /dev/ttyUSB0 9600` connected to the console port on a HP switch but no issues what so ever.

**Comment by EliasA** [comment]
In my case copying the compiled destination fails even when run as sudo. Gives a permission denied error. Ubuntu 16.04, 4.4.0-91-generic #114-Ubuntu SMP

**Comment by EliasA** [comment]
Yes - the compilation process has to be done as root, not with sudo. Alas - sudo -i before compiling. Thank you for contributors! Moxa support for Linux seems to bee bad.

**Comment by tumoxep** [comment]
Successfully compiled on 4.4.0-98-generic #121-Ubuntu SMP. Stops working at 4.4.0-101. Installing kernel sources won't fix it.
